src/iarc7_motion/iarc_tasks/velocity_task.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
```

Comment:
I was hoping that a time could be set for this task to maintain the velocity for then exit.